apps/condo/domains/notification/constants/constants.js

```diff
@@ -950,6 +967,9 @@ const HUAWEI_APP_TYPE_BY_APP_ID = {
 const DEFAULT_TEMPLATE_FILE_EXTENSION = 'njk'
 const DEFAULT_TEMPLATE_FILE_NAME = `default.${DEFAULT_TEMPLATE_FILE_EXTENSION}`
 
+const DEFAULT_TITLE_FOR_NEW_MOBILE_APP_VERSION_PUSH_NOTIFICATION = 'Пора обновиться 😉'
```

Comment:
Hey, bro, we don't use this language here :)
Use `intl` please

Reply:
I'm not sure that's true
https://github.com/open-condo-software/condo/blob/master/apps/condo/domains/resident/constants/constants.js